src/client/src/App.js

```diff
@@ -55,12 +61,17 @@ class App extends Component {
                     type="text"
                     value={this.state.text}
                     onKeyPress={this.onKeyPress}
-                    onChange={e => this.setState({ text: e.target.value })}/>
+                    onChange={e => this.setState({ text: e.target.value },()=> console.log())}/>
             </form>
-            {this.state.conversation.messages.slice().reverse().map((value, index) => {
-                  return <p key={index}><b>{value.sender.name}: </b>{value.text}</p>
-            })}
-          </div>
+
+            <ChatFeed
+              messages={[new Message({id: 1,message: this.state.conversation.lastMessage.text,senderName:this.state.user.name}),
```

Comment:
Hmm, so looks like this kind of changes the spec.

I would imagine that breaks in conversation create a new chat bubble. Instead of only have a single chat bubble per user

Reply:
Also we should clarify if a chat session only allows 2 users max

Reply:
what if we only allow 2 users for now, for simplicity sake. and we can add group chat functionality in later version.

Reply:
> Hmm, so looks like this kind of changes the spec.
> 
> I would imagine that breaks in conversation create a new chat bubble. Instead of only have a single chat bubble per user

do you mean each word would occupy one bubble, and as user type each word, more bubbles will pop out?

Reply:
Sounds good on 2 users. Spun up an extra ticket to cover error handling (e.g. a 3rd user tries to join): https://github.com/imann24/now/issues/17

Let's see Issue #17 is outside the scope of this PR though and tackle that later.

In terms of bubbles, my thought is: create new bubbles whenever the person talking changes. For example:

Isaiah is talking (bubble 1 - blue)
Bill starts talking (bubble 2 - white)
Isaiah resumes talking (bubble 3 - blue)
Bill resumes talking (bubble 4 - white)

What do you think?